### PR TITLE
[feature] Added help command

### PR DESCRIPTION
**Pull request by Xiaro**

### CODE REVIEW

File: src/main/kotlin/org/kamiblue/botkt/command/commands/HelpCommand.kt [hunk 1]
```diff
@@ -0,0 +1,51 @@
+package org.kamiblue.botkt.command.commands
+
+import org.kamiblue.botkt.Main
+import org.kamiblue.botkt.command.BotCommand
+import org.kamiblue.botkt.utils.Colors
+
+object HelpCommand : BotCommand(
+    name = "help-mod",
```
Comment (l1ving) [inline src/main/kotlin/org/kamiblue/botkt/command/commands/HelpCommand.kt:8]: default name should be "commands", and add an alias for "help" and "help-mod"

**Review by l1ving — CHANGES_REQUESTED** [review]

File: src/main/kotlin/org/kamiblue/botkt/command/commands/HelpCommand.kt (hunk 1 above)
Comment (Xiaro) [reply src/main/kotlin/org/kamiblue/botkt/command/commands/HelpCommand.kt:8]: “help" is gonna be conflicting with the js bot. And it is HelpCommand not "CommandsCommand"

Comment (l1ving) [reply src/main/kotlin/org/kamiblue/botkt/command/commands/HelpCommand.kt:8]: I know it'll conflict, I'll rename the bot-js one

Comment (l1ving) [reply src/main/kotlin/org/kamiblue/botkt/command/commands/HelpCommand.kt:8]: bot-kt is a replacement for bot-js

Comment (Xiaro) [reply src/main/kotlin/org/kamiblue/botkt/command/commands/HelpCommand.kt:8]: 🗿

**Review by l1ving — APPROVED** [review]
lgtm, thanks

**Pull request merged**